code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -273,17 +320,30 @@ There are several things that need to be remembered:
 
 	if(client && hud_used)
 		var/atom/movable/screen/inventory/inv = hud_used.inv_slots[TOBITSHIFT(ITEM_SLOT_FEET) + 1]
-		inv.update_appearance()
+		inv.update_icon()
 
 	if(shoes)
-		shoes.screen_loc = ui_shoes //move the item to the appropriate screen loc
-		if(client && hud_used?.hud_shown)
-			if(hud_used.inventory_shown) //if the inventory is open
-				client.screen += shoes //add it to client's screen
-		update_observer_view(shoes,1)
-		overlays_standing[SHOES_LAYER] = shoes.build_worn_icon(default_layer = SHOES_LAYER, default_icon_file = 'icons/mob/clothing/feet.dmi')
-		var/mutable_appearance/shoes_overlay = overlays_standing[SHOES_LAYER]
-		if(OFFSET_SHOES in dna.species.offset_features)
+		var/obj/item/worn_item = shoes
+		var/mutable_appearance/shoes_overlay
+		var/icon_file
+		update_hud_shoes(worn_item)
+		var/handled_by_bodytype = TRUE
+
+		//(Currently) unused digitigrade handling
+		/*if((dna.species.bodytype & BODYTYPE_DIGITIGRADE) && (worn_item.supports_variations & DIGITIGRADE_VARIATION))
+			var/obj/item/bodypart/leg = src.get_bodypart(BODY_ZONE_L_LEG)
+			if(leg.limb_id == "digitigrade")//Snowflakey and bad. But it makes it look consistent.
+				icon_file = DIGITIGRADE_SHOES_PATH*/
```

Comment:
Does this cause error sprites if commented? I have a feeling it shouldn't

Reply:
No but it's a micro optimization to leave it commented if its unused.